api/src/main/java/jakarta/data/metamodel/Expression.java

```diff
@@ -46,13 +46,11 @@ default Restriction<T> in(Set<V> values) {
         return BasicRestriction.of(this, In.values(values));
     }
 
-    default Restriction<T> in(
-            @SuppressWarnings("unchecked") V... values) {
+    default Restriction<T> in(@SuppressWarnings("unchecked") V... values) {
```

Comment:
Do these warning suppressions actually work for you, @otaviojava? They don't work for me, at least not when they're placed on the parameter.

Reply:
It did not work for me either.

Reply:
Indeed, my IDE recommends that I remove those annotations.

Reply:
OK, thx.